lib/recurly/external_product.rb

```diff
@@ -0,0 +1,20 @@
+module Recurly
+  class ExternalProduct < Resource
+
+    # @return [Plan]
+    belongs_to :plan
+
+    # @return array [ExternalProductReference]
+    has_many :external_product_references
+
+    define_attribute_methods %w(
+      name
+      created_at
+      updated_at
+    )
```

Comment:
in order to get the `plan` to show up correctly with the `plan_code` and `name`, you need to do something similar to what we do in `lib/recurly/subscription.rb`